R/sem.R

```diff
@@ -462,33 +462,76 @@ checkLavaanModel <- function(model, availableVars) {
 
   modelContainer[["fittab"]] <- fittab
 
+  if (options[["group"]] != "") {
+    grouptab <- createJaspTable(title = gettext("Model fit by group"))
+    grouptab$dependOn("models")
+    grouptab$position <- 0.05
+
+    grouptab$addColumnInfo(name = "Model",    title = "",                            type = "string" , combine = TRUE)
+    grouptab$addColumnInfo(name = "group",    title = gettext("Group"),              type = "string" )
+    grouptab$addColumnInfo(name = "AIC",      title = gettext("AIC"),                type = "number" )
+    grouptab$addColumnInfo(name = "BIC",      title = gettext("BIC"),                type = "number" )
+    grouptab$addColumnInfo(name = "N",        title = gettext("n"),                  type = "integer")
+    grouptab$addColumnInfo(name = "Chisq",    title = gettext("&#967;&sup2;"),       type = "number" ,
+                         overtitle = gettext("Baseline test"))
+    grouptab$addColumnInfo(name = "Df",       title = gettext("df"),                 type = "integer",
+                         overtitle = gettext("Baseline test"))
+    grouptab$addColumnInfo(name = "PrChisq",  title = gettext("p"),                  type = "pvalue",
+                         overtitle = gettext("Baseline test"))
+    if (length(options[["models"]]) > 1) {
+      grouptab$addColumnInfo(name = "dchisq",   title = gettext("\u0394\u03C7\u00B2"), type = "number" ,
+                           overtitle = gettext("Difference test"))
+      grouptab$addColumnInfo(name = "ddf",      title = gettext("\u0394df"),           type = "integer",
```

Comment:
here we only want to translate the "df"
```suggestion
      grouptab$addColumnInfo(name = "ddf",      title = gettextf("%1$sdf", "\u0394"),           type = "integer",
```